environments/common/configmaps/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ configMapGenerator:
   - |
     CORS_ORIGINS=["*"]
   - |
-    API_SERVICES=["boilerplate", "query", "variant", "meta", "gene"]
+    API_SERVICES=["boilerplate", "query", "variant", "meta", "gene", "patient"]
```

Comment:
What about the HPO service?

Reply:
I'm not aware of it. It is not orchestrated right now so it is not running (not as a separate service anyways).

If you need it to run now, let me know and I'll add it to the orchestration.

Btw, this parameter is a legacy thing from the way the repo is structured. It will be eventually be phased out.
@jecos

Reply:
Yes, we need the HPO service now

Reply:
I'll add the orchestration for HPO in a separate PR